<!-- -->
<launch>
<!-- *********************************** -->
<!--         CONFIG               -->
<!-- *********************************** -->
<arg name="map_frame"               default="map"/>
<arg name="robot_frame"             default="base_footprint"/>
<arg name="move_base_pose_topic"    default="/move_base_simple/goal"/>
<arg name="req_path_topic"          default="/request_path"/>
<arg name="brick_found_topic"       default="/brick_found"/>

<arg name="path_rate"               default="2.0"/>
<arg name="comp_x"                  default="0.2"/>
<arg name="comp_y"                  default="0.2"/>
<arg name="comp_yaw"                default="0.523599"/>

<!-- *********************************** -->
<!--                NODES                -->
<!-- *********************************** -->

<!-- Explore Move -->
<node name="ExploreMove"           pkg="explore_move" type="explore_move_node"  output="screen">
    <param name="map_frame"             type="string"       value="$(arg map_frame)"/>
    <param name="robot_frame"           type="string"       value="$(arg robot_frame)"/>
    <param name="move_base_pose_topic"  type="string"       value="$(arg move_base_pose_topic)"/>
    <param name="req_path_topic"        type="string"       value="$(arg req_path_topic)"/>
    <param name="brick_found_topic"     type="string"       value="$(arg brick_found_topic)"/>
    <param name="path_rate"             type="double"       value="$(arg path_rate)"/>
    <param name="comp_x"                type="double"       value="$(arg comp_x)"/>
    <param name="comp_y"                type="double"       value="$(arg comp_y)"/>
    <param name="comp_yaw"              type="double"       value="$(arg comp_yaw)"/>

</node>

</launch>